page-objects/pages/TASKPage.js

```diff
@@ -14,13 +14,20 @@ const remoteFilePathInput = 'input[type=file][class=dz-hidden-input]';
 const commentBtn = '#comment-tab-link';
 const commentGradingTabSel = '#feedback-tab-link';
 const hometasksTabSel = 'button[data-testid="hometasks"]';
+const nuxtHometasksTabSel = '[data-testid="Offene Aufgaben"] > .link-name';
 const taskRatingInput = '[data-testid="evaluation_procent"]';
 const ratingViewSel = '.grade';
 const remarkViewSel = '.ckcontent.comment';
 const submissionsTable = '#submissions table';
 const submissionRow = `${submissionsTable} tbody tr.userinfo`;
+const withoutDueDateTask = '.v-expansion-panel--next-active > .v-expansion-panel-header';
+const completedTaskTab = '[aria-selected="false"]';
```

Comment:
const withoutDueDateTask and const completedTaskTab are bad selectors.
Ask your team, or we can try it ourselfs, to create data-testIds for those panels.

Reply:
Absolutely agree about the selectors. Ideally we have to put data-testids or at least take those selectors which are unique.